src/main/java/org/umcn/me/output/ReadCounts.java

```diff
@@ -5,45 +5,64 @@
 import org.omg.CORBA.DynAnyPackage.InvalidValue;
 import org.umcn.me.pairedend.MobilePrediction;
 import org.umcn.me.samexternal.SAMSilentReader;
+import org.umcn.me.util.ArrayStats;
 import org.umcn.me.util.MobileDefinitions;
 
 import java.io.File;
 import java.io.IOException;
-import java.util.List;
-import java.util.Properties;
-import java.util.Vector;
+import java.util.*;
 
-public class VAFPredictions {
+public class ReadCounts {
 
-    public static Logger logger = Logger.getLogger(VAFPredictions.class.getName());
+    public static Logger logger = Logger.getLogger(ReadCounts.class.getName());
+    public static boolean VAF = false;
 
     public static Vector<MobilePrediction> runFromProperties(Properties props, Vector<MobilePrediction> predictions) throws IOException {
 
-        //Don't predict the VAFs when no index file is available
+        //Don't predict the VAFs if the index files aren't available
         String samIndexPath = props.getProperty(MobileDefinitions.INFILE_INDEX);
-        if(samIndexPath == null)
-            return predictions;
+        VAF = samIndexPath != null;
 
-        logger.info("Calculating VAF of predictions based on the maximum ratio of clipped + discordant / total reads within " + props.getProperty(MobileDefinitions.VAF_DETERMINATION_WINDOW) + "bp to the left and right of the insertion");
-        for (MobilePrediction prediction : predictions) {
-            int vafWindow = Integer.parseInt(props.getProperty(MobileDefinitions.VAF_DETERMINATION_WINDOW).trim());
-
-            //Load the bam file
-            File sam = new File(props.getProperty(MobileDefinitions.INFILE));
-            File index = new File(samIndexPath);
-            SAMSilentReader samFile = new SAMSilentReader(sam, index);
+        logger.info("Determining the number of non-supporting reads within the clusters for each prediction");
+        if(VAF) logger.info("and calculating the VAF based on the maximum ratio of clipped + discordant / total reads within " + props.getProperty(MobileDefinitions.VAF_DETERMINATION_WINDOW) + " bp to the left and right of the insertion");
```

Comment:
as a style tip, is line is very wide, lints will usually complain about this
a multi-line if reads easier